### DISCUSSION TITLE
2412.1 Available and opening up Discussions on the repo

### DISCUSSION THREAD

**Discussion by rbalsleyMSFT** [rbalsleyMSFT/FFU#114] [Announcements]
I released 2412.1 today, which was the biggest released we've had with a lot of quality of life improvements and support for Windows Server. I'd highly recommend updating today.

I'm also turning on discussions today. This is a better avenue to get feedback instead of opening Github issues.

**Comment by HedgeComp** [comment]
This is great idea. thanks @rbalsleyMSFT  for setting this up.

**Comment by RonarPS** [comment]
Nice. Will have a look at it in the next days. May also have a new feature request😁 will sum it up, and repost it here

**Comment by wildcard0063** [comment]
The FFU file was not found after adding the apps to the VM.  I have attached the log file.
[FFUDevelopment.log](https://github.com/user-attachments/files/18386560/FFUDevelopment.log)

**Reply by rbalsleyMSFT** [reply]
Let's move this discussion to this issue https://github.com/rbalsleyMSFT/FFU/issues/126
